217.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @param {number[]} nums
+ * @return {boolean}
+ */
+var containsDuplicate = function (nums) {
+  nums.sort((a, b) => a - b);
```

Comment:
This can cause side effects, which you should avoid.
To avoid them, you can create a copy of `nums` and sort it. 

```suggestion
  const sorted = [...nums].sort((a, b) => a - b);
```